docs/guide/helpers.md

```diff
@@ -6,7 +6,19 @@ parent: Guide
 
 # Helpers
 
-Helper methods can be used through the `helpers` proxy:
+Helpers must be included in order to be used:
+
+```ruby
+class UserComponent < ViewComponent::Base
+  include IconHelper
```

Comment:
With this order flipping, you may want to define/introduce the `IconHelper` module in this section (as opposed to below in the `helpers` proxy recommendation).

Reply:
Oh, yes! Thanks for catching this ❤️